sesame/src/main/scala/org/w3/banana/sesame/SesameModule.scala

```diff
@@ -38,6 +38,8 @@ trait SesameModule
 
   implicit val rdfStore: RDFStore[Sesame, Try, RepositoryConnection] with SparqlUpdate[Sesame, Try, RepositoryConnection] = new SesameStore
 
+  implicit val transactor: Transactor[Sesame, RepositoryConnection] = rdfStore
```

Comment:
I have to resolve transactor implicitly in my transactor test. If you want to I can resolve a store there instead.